src/javascript/app_2/App/Components/Elements/PositionsDrawer/positions_drawer.jsx

```diff
@@ -0,0 +1,59 @@
+import classNames                     from 'classnames';
+import PropTypes                      from 'prop-types';
+import React                          from 'react';
+import { Scrollbars }                 from 'tt-react-custom-scrollbars';
+import { localize }                   from '_common/localize';
+import { IconMinus }                  from 'Assets/Common';
+import { connect }                    from 'Stores/connect';
+
+class PositionsDrawer extends React.Component {
+    componentDidMount()    { this.props.onMount(); }
```

Comment:
we don't really write functions as one-liners, would be better to move the contents into a second line